Scripts/lint.sh

```
@@ -1,45 +1,93 @@
+run_command $MINT_RUN swiftlint lint $SWIFTLINT_OPTIONS
+run_command $MINT_RUN swift-format lint --recursive --parallel $SWIFTFORMAT_OPTIONS Sources Tests
+
+if [ -z "$CI" ]; then
+    run_command $MINT_RUN periphery scan $PERIPHERY_OPTIONS --disable-update-check
 fi
 
-$MINT_RUN periphery scan 
-$MINT_RUN swiftformat --lint $SWIFTFORMAT_OPTIONS .
-$MINT_RUN swiftlint lint $SWIFTLINT_OPTIONS
 
 popd
```

Shellcheck (0.11.0)

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)